commerce_option.info.yml

```diff
@@ -3,3 +3,5 @@ description: This module allows store managers to offer products with multiple o
 type: module
 package: Commerce (contrib)
 core: 8.x
+dependencies:
+  - commerce:commerce
```

Comment:
Please add commerce_product as a dependency as well.

Reply:
Once we add the commerce_product dependency, we should be good to create a PR on D.org and commit and merge it.

Reply:
I added the product dependency somewhere else I guess and avoided adding it here since the branches depend on each other and we can get conflicts
you can see it => https://github.com/krystalcode/drupal8-commerce-option/blob/1402c00776673b61eea6c5c8778a80260d9993bf/commerce_option.info.yml

Reply:
Ah, okay then. Everything looks good. Good to create the PR on d.org and I'll RTBC it.